src/core_modules/capture-core/components/Pages/common/TEIRelationshipsWidget/TeiSearch/serverToFilters.js

```diff
@@ -0,0 +1,38 @@
+// @flow
```

Comment:
Hey @eirikhaugstulen,
I recently merged some small changes into the [serverToFilters.js](https://github.com/dhis2/capture-app/commits/master/src/core_modules/capture-core/components/TeiSearch/serverToFilters.js) file. Can you update the relationships widget as well?
Thank you!